blocks/header/header.js

```diff
@@ -177,12 +178,30 @@ async function fetchSearchData() {
       throw new Error(`Fetching search data failed with: ${resp.status}`);
     }
   }
-  return window.searchData;
+  return window.searchData.data;
+}
+
+async function fetchProductSupportSearchData() {
+  if (!window.productSearchData) {
+    const language = window.location.pathname.substring(1, window.location.pathname.indexOf('/', 1));
+    const products = await getProducts(language);
+    window.productSearchData = products.flatMap((product) => ([{
```

Comment:
_cosmetics_: for readability, you could use destructuring for the prouduct and asset objects:

```
     window.productSearchData = products.flatMap(({Name, Description, ProductCodes, assets}) => ([
       {
         title: Name,
         description: Description,
         path: `/${language}/product-support/${ProductCodes.split('|')[0]}`,
       },
       ...assets.map(({Name, Description, URL}) => ({
         title: Name,
         description: Description,
         path: URL,
       }))
     ])); 
```